index.html

```diff
@@ -113,6 +113,33 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card END ________  -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Himanshu Dhir</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://twitter.com/home" target="_blank">Himanshu54276066</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/himanshurd" target="_blank">himanshurd</a>
+          </p>
+          <p class="about">about you</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://www.linkedin.com/feed/?trk=guest_homepage-basic_nav-header-signin" target="_blank" title="First Resource">This is my Linkdin's Account</a>
+              </li>
+              <li>
+                <a href="" target="_blank" title="Second Resource">My hobby is to play cricket. I love to learn more about AI using Python</a>
+              </li>
+              <li>
+                <a href="https://docs.google.com/document/d/1h1GPLL6-ZqLoEjs_Di_eK6iLeSqhIb5LZUcSHLei9Gs/edit?usp=sharing" target="_blank" title="Third resource">This is my resume shows what Projects I have done</a>
```

Comment:
Hey -- friendly warning: you're including your entire personal address and personal phone number in this document. This document will be read by _**millions**_ of people and countless bots scraping for information.

I would like to request that you do not link to a doc that exposes you like this.

Also, professionally speaking as someone who has reviewed hundreds of resumes in this industry... you don't _want_ to put that on there. An employer might decide that you're too far away (even if you would have been okay with the distance), and they're more likely to email back to your application to schedule a call than they are to use the number. Just put the largest city you're close to in the state you're in, and add a professional-sounding email. 😉